core/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -570,7 +581,7 @@ public void testSegmentsStats() {
 
     public void testAllFlags() throws Exception {
         // rely on 1 replica for this tests
-        assertAcked(prepareCreate("test1").setSettings("index.mapping.single_type", false));
+        assertAcked(prepareCreate("test1").setSettings("index.version.created", Version.V_5_6_0.id));
```

Comment:
Can you add a comment about why it is important that we have indices with multiple types?